taskcluster/fenix_taskgraph/pushapk.py

```diff
@@ -9,43 +9,72 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from taskgraph.transforms.base import TransformSequence
-from taskgraph.util.treeherder import inherit_treeherder_from_dep
+from taskgraph.util.treeherder import inherit_treeherder_from_dep, join_symbol
 
 
 transforms = TransformSequence()
 
 
-# TODO remove this transform once signing task are migrated to taskgraph
```

Comment:
These changes to push-apk are necessary because we need it to digest signing tasks coming from `taskgraph`. I don't expect any further changes (during the migration) once this PR is merged